syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -765,6 +700,19 @@ def mean(self, shares, **kwargs):
 
         return result
 
+    @staticmethod
+    def share_combine(tensors_shares):
+        workers = tensors_shares[0].keys()
+
+        return {
+            worker: [tensor_shares[worker] for tensor_shares in tensors_shares]
+            for worker in workers
+        }
+
+    @staticmethod
+    def apply_to_share(op, shares: dict):
```

Comment:
I'm not convinced we gain in readbility: i would remove this function, it's a one liner anyway